### ISSUE TITLE
We should ship new version

### ISSUE THREAD

**Issue by rmhrisk** [PeculiarVentures/tl-create#50]

**Comment by andrewholsted** [comment]
+1 - received an error installing from NPM for the Mozilla bundle... Going to try master now

```
yarn run v1.21.1
$ ./bin/update-certs.sh
Parsing started: 2020:03:10:08:30:19
Trust Lists: Mozilla
Error: Can not parse row 167: CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
Cannot fetch any Trust Lists.
error Command failed with exit code 1.
```

**Comment by andrewholsted** [comment]
Confirmed the same command

node src/bin/tl-create.js --mozilla --format pem roots.pem

Works on master. Any ETA on publishing a new version?

**Comment by rmhrisk** [comment]
@microshine please ship new version

**Comment by microshine** [comment]
@andrewholsted I updated and published a new version of `tl-create@1.3.6-next.1`. Could you try it?

### Install
```
npm i tl-create@next -g
```

### Run
```
> tl-create --mozilla --format pem roots.pem
Parsing started: 2020:3:11:2:35:2
Trust Lists: Mozilla
Output format: PEM
```
